Resources/public/css/theme/views/subitemlist.css

```diff
@@ -92,3 +92,77 @@
     left: 50%;
     position: absolute;
 }
+
+.ez-view-subitemlistview .ez-subitem-hovered-priority-cell  .ez-subitem-priority-icons:after {
+    content: "\e606";
+    background: #E6E6E6;
+    color: #737373;
+    border: 1px solid #888;
+}
+
+.ez-view-subitemlistview .ez-subitem-error-priority-cell  .ez-subitem-priority-icons:after {
+    content: "!";
+    background: #BF3E33;
+    color: #fff;
+    border: 1px solid #BF3E33;
+}
+
+.ez-view-subitemlistview .ez-subitem-priority-icons:after {
+    font-weight: bold;
+}
+
+.ez-view-subitemlistview td.ez-subitem-error-priority-cell,
+.ez-view-subitemlistview .ez-subitemlist-table tr:nth-child(2n-1) td.ez-subitem-error-priority-cell {
+    background: #FEECEA;
+    box-shadow: none;
+    border: 1px double #BF3E33;
+}
+
+.ez-view-subitemlistview .ez-subitem-selected-priority-cell {
+    box-shadow: none;
+    border: 1px double #888;
+}
+
+.ez-view-subitemlistview input.ez-subitem-priority-input,
```

Comment:
kind of the same as https://github.com/ezsystems/PlatformUIBundle/pull/527/files#r57181559: the tag name `input` should be removed and you should change the rules in theme/app.css related to form elements (input, textarea, button) to:

1. not be prefixed with `.ez-platformui-app` (this was made a long time ago for a now invalid reason)
2. put those CSS rules in theme/modules/form.css